services/venice-controller/src/main/java/com/linkedin/venice/ingestion/control/RealTimeTopicSwitcher.java

```diff
@@ -145,8 +145,7 @@ void ensurePreconditions(
      */
     createRealTimeTopicIfNeeded(store, version, srcTopicName, hybridStoreConfig.get());
     if (version != null && version.isSeparateRealTimeTopicEnabled()) {
-      PubSubTopic separateRealTimeTopic =
-          pubSubTopicRepository.getTopic(Version.composeSeparateRealTimeTopic(store.getName()));
+      PubSubTopic separateRealTimeTopic = pubSubTopicRepository.getTopic(Utils.getRealTimeTopicName(store));
```

Comment:
We should get `isSeparateRealTimeTopicEnabled` topic instead  `getRealTimeTopicName` of  right?